### DISCUSSION TITLE
Log storage duration for link tracking

### DISCUSSION THREAD

**Discussion by dxygit1** [apache/skywalking#12292] [Q&A]
I want to adjust the retention period of the log for link tracing, but I don't know the default value and which parameters need to be adjusted.

**Comment by wu-sheng** [comment]
I don't know what is your asking.
Maybe TTL is the thing, https://skywalking.apache.org/docs/main/latest/en/setup/backend/ttl/

**Reply by dxygit1** [reply]
recordDataTTL means three days by default?

**Reply by dxygit1** [reply]
This is the tracking log retention time, right?

**Reply by Superskyyy** [reply]
record covers logs yes

**Comment by dxygit1** [comment]
![img_v3_02bh_411b924e-c3ed-48bf-83e1-68b41b75968g](https://github.com/apache/skywalking/assets/57128304/37d1bb61-b8f5-454f-956c-c2b9c5ed890e)
This is the retention period for the tracking logs here.

**Reply by wu-sheng** [reply]
You are using a historic version, 4 or 5 years ago. No interest to talk about that.

**Reply by dxygit1** [reply]
Okay, thank you.
